<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>spec_options</class>
 <widget class="QDialog" name="spec_options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>321</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Создание спектра зарядов в импульсе</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>270</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="charge_param_table">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>251</width>
     <height>121</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>281</width>
     <height>16777215</height>
    </size>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <row>
    <property name="text">
     <string>Заряд, Кл</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Значение</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
  </widget>
  <widget class="QComboBox" name="spectrum_box">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Постоянный заряд</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Равномерный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Гауссовский</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Линейчатый</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Экспоненциальный</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="spectrum_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Спектр зарядов</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="spectrum_square">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCustomPlot" name="spec_plot" native="true">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>351</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>spec_options</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>spec_options</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
